packages/libnvidia-container/libnvidia-container.spec

```diff
@@ -0,0 +1,78 @@
+%global nvidia_modprobe_version 450.57
+
+Name: %{_cross_os}libnvidia-container
+Version: 1.5.0
+Release: 1%{?dist}
+Summary: NVIDIA container runtime library
+License: Apache-2.0 and GPL-2.0-only
```

Comment:
Another license to double check. The other licenses may be for vendored dependencies we don't use?

```
❯ askalono crawl .
./COPYING
License: GPL-3.0-only (original text)
Score: 0.998
./COPYING.LESSER
License: LGPL-3.0-only (original text)
Score: 0.998
./NOTICE
Error: Confidence threshold not high enough for any known license
./LICENSE
License: Apache-2.0 (original text)
Score: 0.999
```

It should be aligned with the license files we're including.